liberty-maven-plugin/src/main/resources/META-INF/plexus/components.xml

```diff
@@ -20,17 +20,47 @@
                     <lifecycle>
                         <id>default</id>
                         <phases>
+                            <pre-clean>
+                                net.wasdev.wlp.maven.plugins:liberty-maven-plugin:stop-server
+                            </pre-clean>
                             <process-resources>
-                                org.apache.maven.plugins:maven-resources-plugin:resources,
-                                net.wasdev.wlp.maven.plugins:liberty-maven-plugin:create-server
+                                org.apache.maven.plugins:maven-resources-plugin:3.0.2:resources
```

Comment:
Do we have to include the version here?  I am sure it can be overridden in the pom or use the default version in the super pom if not specified.  Adding a version here, does that mean we have to maintain it when newer version is available.